src/iss/Location.js

```diff
@@ -0,0 +1,50 @@
+/* @flow */
+import _ from "underscore";
+import { titleize } from "underscore.string";
+
+export default class Location {
+    props: issLocation;
+
+    constructor(props: issLocation) {
+        this.props = props;
+    }
+
+    googleMapsUrl(): string {
+        var query = encodeURIComponent(
+            `${this.streetAddressLine1()} ${this.streetAddressLine2()}`
+        );
+
+        return `https://maps.google.com/?q=${query}`;
+    }
+
+    streetAddressLine1(): string {
+        var addrDescriptors = [
+            this.props.flat_unit,
+            this.props.level,
+            this.props.building,
+        ].map(text => text.trim());
+
+        addrDescriptors = _(addrDescriptors)
+            .compact()
+            .map(part => `${part}, `)
+            .join("");
```

Comment:
If you want a comma between `addrDescriptors` and `street` just finish with `[a, b].compact().join(', ')`